packages/mdc-menu/_mixins.scss

```diff
@@ -20,12 +20,128 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
+@import "@material/rtl/mixins";
+@import "@material/list/mixins";
+@import "@material/ripple/mixins";
 @import "./variables";
 
+
+@mixin mdc-menu($query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-ripple-common($query);
+
+  // Customize the menu-surface and contained list to match the mdc-menu styles.
+  .mdc-menu {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-list-item-meta-ink-color(text-primary-on-background);
+      @include mdc-list-item-graphic-ink-color(text-primary-on-background);
+    }
+
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-menu-base_;
+
+      .mdc-list-divider {
+        @include mdc-list-divider_;
+      }
+
+      .mdc-list-item {
+        @include mdc-list-item_;
+      }
+
+      .mdc-list-item--disabled {
+        @include mdc-list-item--disabled_;
+      }
+
+      //stylelint-disable selector-max-type, selector-no-qualifying-type
+      a.mdc-list-item .mdc-list-item__text,
+      a.mdc-list-item .mdc-list-item__graphic {
+        @include mdc-list-item-link-content_;
+      }
+      // stylelint-enable selector-max-type, selector-no-qualifying-type
+    }
+  }
+
+  // postcss-bem-linter: define menu
+  .mdc-menu__selection-group {
+    @include mdc-feature-targets($feat-structure) {
```

Comment:
I might be inclined to leave this as-is, otherwise it might encourage having multiple instances of the same selector within separate `feature-targets` blocks down the road. (See https://github.com/material-components/material-components-web/pull/4279/files/12a162a9e3925f7e0a482387677f08a485a72571#r250810168 for an example.)